plugins/samba.lua

```diff
@@ -0,0 +1,130 @@
+if lldap == nil then
+    lldap = {}
+end
+
```

Comment:
What's that for?

Reply:
Just my syntax highlighter, don't plan to keep it in :-)